age/keysource.go

```diff
@@ -96,27 +96,46 @@ func (key *MasterKey) SetEncryptedDataKey(enc []byte) {
 
 // Decrypt decrypts the EncryptedKey field with the age identity and returns the result.
 func (key *MasterKey) Decrypt() ([]byte, error) {
-	ageKeyFilePath, ok := os.LookupEnv("SOPS_AGE_KEY_FILE")
+	var ageKeyReader io.Reader
+	var ageKeyReaderName string
+
+	if ageKeyReader == nil {
+		ageKey, ok := os.LookupEnv("SOPS_AGE_KEY")
```

Comment:
I know it wasn't this way before, but I think we should move `SOPS_AGE_KEY` and `SOPS_AGE_KEY_FILE` to constants.